tests/level1a/handlers/test_level1a.py

```diff
@@ -220,3 +249,51 @@ def test_interpolate():
             index=datetimes,
         )
     )
+
+
+@patch("level1a.handlers.level1a.pa.fs.S3FileSystem", return_value=None)
+@patch.dict(os.environ, {
+    "OUTPUT_BUCKET": TemporaryDirectory().name,
+    "PLATFORM_BUCKET": str(Path(__file__).parent / "files" / "platform"),
+    "HTR_BUCKET": str(Path(__file__).parent / "files" / "rac"),
+})
+def test_lambda_handler(patched_s3):
+    out_dir = os.environ["OUTPUT_BUCKET"]
+    (Path(out_dir) / "2022" / "11" / "22").mkdir(parents=True)
```

Comment:
These folders are created automatically when working with S3. That doesn't happen here, hence the need for manual creation.